onpay.php

```diff
@@ -425,10 +470,37 @@ private function getPaymentWindow($order, $payment, $currency) {
         return $paymentWindow;
     }
 
+    /**
+     * Method used for setting a payment info parameter. The value is attempted set, if this fails we'll ignore the value and do nothing.
+     * $value can be a single value or an array of values passed on as arguments.
+     * Validation of value happens directly in the SDK.
+     *
+     * @param $paymentInfo
+     * @param $parameter
+     * @param $value
+     */
+    private function setPaymentInfoParameter(&$paymentInfo, $parameter, $value) {
```

Comment:
There is no need to pass $paymentInfo as a reference, it will already be a reference to the object you get in.

https://www.php.net/manual/en/language.oop5.references.php